activemq-transport/src/main/java/com/esri/geoevent/transport/activemq/ActiveMQOutboundTransportDefinition.java

```diff
@@ -69,7 +69,7 @@ public String getDomain() {
 
   @Override
   public String getVersion() {
-    return "10.4.1";
+    return "10.3.0";
```

Comment:
`10.3.0` or `10.4.1`? The versions need to match the component (transport) version.